server/src/main/java/org/opensearch/index/SegmentReplicationShardStats.java

```diff
@@ -134,6 +135,7 @@ public void writeTo(StreamOutput out) throws IOException {
         out.writeVLong(currentReplicationTimeMillis);
         out.writeVLong(lastCompletedReplicationTimeMillis);
         out.writeVLong(currentReplicationLagMillis);
+        out.writeOptionalWriteable(currentReplicationState);
```

Comment:
this object is set at the coordinator to format the rest response, you shouldn't need to send this over the wire?

Reply:
I see. I found `currentReplicationState` is missing  earlier and set because were setting the `currentReplicationState` at shard level earlier. Now we dont need it.